### PR TITLE
Add MVB case study

### PR DESCRIPTION
**Pull request by marcoow**
This adds the case study page for our project with MVB:

[Preview](https://deploy-preview-1920--objective-northcutt-37494c.netlify.app/cases/mvb/)

localhost_8080_cases_mvb_

see #1914

### CODE REVIEW

**Review by SaeLLo — COMMENTED** [review]
Looks good.

**Review by SaeLLo — COMMENTED** [review]
Header: There should be spaces between the slashes (See other case studies)

**Comment by marcoow** [comment]
good catch – fixed 👍

File: src/cases/mvb.md [hunk 1]
```diff
@@ -1,10 +1,73 @@
 ---
 layout: case-study
-company: mvb
+company: MVB
+title: A platform for the german book industry | Work
+displayTitle: "A platform for the <em>german</em> book industry"
 description:
   <p>mvb provides marketing solutions for the german book industry.</p><p>They
```
Comment (emmasofiah2o) [inline src/cases/mvb.md]:
```suggestion
  <p>mvb provides marketing solutions for the German book industry.</p><p>They
```

File: src/cases/mvb.md [hunk 2]
```diff
@@ -1,10 +1,73 @@
...
   approached Mainmatter when they were looking for external expertise. We
   performed a workshop, leveling up the team’s expertise and guided the project
```
Comment (emmasofiah2o) [inline src/cases/mvb.md:9]:
```suggestion
  performed a workshop, leveling up the team’s knowledge and guided the project
```

File: src/cases/mvb.md [hunk 3]
```diff
@@ -1,10 +1,73 @@
...
+  <div class="case-study__text">
+    <h3 class="h4">MVB (part of the Börsenverein Group) provides digital platforms and services for the marketing and distribution of books for publishing houses and bookstores in Germany and abroad. One of their core offerings is VLB-TIX which provides a database with detailed information on new titles of all 21.000 publishers in Germany.</h3><br>
+    <p>The platform improves the flow of information between publishing houses, booksellers, journalists, bloggers, and interested readers.</p>
+  </div>
+</div>
+
+<div class="case-study__section">
+  <h2 class="case-study__heading h5">The Challenge</h2>
+  <div class="case-study__text">
+    <p>Having previously relied on an external provider to develop VLB-TIX for them, MVB’s goal was in-house development and maintenance of the platform. At the same time, they needed to rewrite the existing product to align it with their internal tech stack and to integrate it better with other systems. Lacking the internal expertise and staff for a strategic project like this, MVB reached out to Mainmatter for help.</p>
```
Comment (emmasofiah2o) [inline src/cases/mvb.md:32]:
```suggestion
    <p>Having previously relied on an external provider to develop VLB-TIX for them, MVB’s goal was in-house development and maintenance of the platform. At the same time, they needed to rewrite the existing product to align it with their internal tech stack and to integrate it better with other systems. Lacking the internal expertise and staff for a strategic project this size, MVB reached out to Mainmatter for help.</p>
```

**Review by emmasofiah2o — COMMENTED** [review]
Also a few suggestions here. This one sounds very professional :)

File: src/cases/mvb.md (hunk 3 above)
Comment (marcoow) [reply src/cases/mvb.md:32]: it's not about size

**Comment by davidhajba** [comment]
Looks good.

**Review by davidhajba — APPROVED** [review]

**Pull request merged**